web/frontend/src/app/component/general-search/general-search.component.html

```diff
@@ -167,7 +167,7 @@
               </ng-template>
               <ng-template pTemplate="rowexpansion" let-rowData let-columns="columns">
                 <tr>
-                  <td [attr.colspan]="columns.length + 1">
+                  <td *ngIf="columns.length > 0" [attr.colspan]="columns.length + 1">
```

Comment:
Peter, just verifying this was an intentional change (and curious what problem it sovles)